src/main/java/frc/robot/subsystems/LiftSubsystem.java

```diff
@@ -34,6 +38,10 @@ public LiftSubsystem(final RobotMap.LiftMap map) {
         lowerLimit = map.getLowerLimit();
         upperLimit = map.getUpperLimit();
         manipAngle = map.getManipAngle();
```

Comment:
This is part of the "maflipulator" subsystem now it should be removed from the lift.